src/App.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 
+import { useCookies } from 'react-cookie';
+import { IntlProvider } from 'react-intl';
+
 import RoutesConfig from 'routes/RoutesConfig';
+import en from 'translations/en.json';
+import fr from 'translations/fr.json';
+import pt from 'translations/pt.json';
 
 import 'styles/global.css';
 
+export const messages = {
+  en,
+  fr,
+  pt,
+};
+
 function App() {
+  const [cookies] = useCookies();
+
   return (
-    <div className="bg-transit-grey w-screen h-screen">
-      <RoutesConfig />
-    </div>
+    <IntlProvider
+      locale={cookies.language.value}
```

Comment:
please use default language when there is no such cookie with saved selected language etc

Reply:
there is error 'Uncaught TypeError: cookies.language is undefined'